other/web-js/client/package.json

```diff
@@ -11,22 +11,29 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
-    "dedent": "^0.7.0",
+    "bson": "^5.0.1",
+    "buffer": "^6.0.3",
+    "crypto-browserify": "^3.12.0",
```

Comment:
can this (and maybe other dependencies) be deleted since moving to vite? 

stream-browserify and buffer perhaps?

Reply:
also the craco build/test commands and dev dependency for craco